frontend/projects/ui/src/app/pages/server-routes/sessions/sessions.page.html

```diff
@@ -8,97 +8,116 @@
 </ion-header>
 
 <ion-content class="ion-padding-top">
-  <!-- loading -->
-  <ion-item-group *ngIf="loading; else notLoading">
-    <div *ngFor="let entry of ['This Session', 'Other Sessions']">
-      <ion-item-divider>{{ entry }}</ion-item-divider>
-      <ion-item style="padding-bottom: 6px">
-        <ion-avatar slot="start" style="margin-right: 30px">
-          <ion-skeleton-text
-            animated
-            style="width: 40px; height: 40px; border-radius: 0"
-          ></ion-skeleton-text>
-        </ion-avatar>
-        <ion-label>
-          <ion-skeleton-text
-            animated
-            style="width: 150px; height: 20px; margin-bottom: 10px"
-          ></ion-skeleton-text>
-          <ion-skeleton-text
-            animated
-            style="width: 250px; height: 14px; margin-bottom: 12px"
-          ></ion-skeleton-text>
-          <ion-skeleton-text animated style="width: 350px"></ion-skeleton-text>
-        </ion-label>
-        <ion-button *ngIf="entry === 'second'" slot="end" fill="clear">
-          <ion-skeleton-text
-            animated
-            style="width: 60px; border-radius: 0"
-          ></ion-skeleton-text>
-        </ion-button>
-      </ion-item>
+  <ion-item-group>
+    <!-- current session -->
+    <ion-item-divider>Current Session</ion-item-divider>
+    <div class="grid-fixed">
+      <ion-grid class="ion-padding" style="min-width: 840px">
+        <ion-row class="grid-headings">
+          <ion-col size="6">User Agent</ion-col>
+          <ion-col size="3">Platform</ion-col>
+          <ion-col size="3">Last Active</ion-col>
+        </ion-row>
+        <!-- loading -->
+        <ng-container *ngIf="loading$ | async; else loaded">
+          <ion-row class="ion-align-items-center grid-row-border">
```

Comment:
Looks like we got this setup quite often, maybe we should make a component `loading-row [number]="x"` to show `x` amount of `ion-row` with a skeleton?